commands/work/work.go

```diff
@@ -0,0 +1,120 @@
+package work
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/taskcluster/slugid-go/slugid"
+	cmd_ep "github.com/taskcluster/taskcluster-worker/commands/extpoints"
+	"github.com/taskcluster/taskcluster-worker/config"
+	"github.com/taskcluster/taskcluster-worker/engines/extpoints"
+	"github.com/taskcluster/taskcluster-worker/runtime"
+	"github.com/taskcluster/taskcluster-worker/worker"
+)
+
+func init() {
+	cmd_ep.CommandProviders.Register(cmd{}, "work")
+}
+
+type cmd struct{}
+
+func (cmd) Summary() string {
+	return "Start the worker."
+}
+
+func (cmd) Usage() string {
+	return `Usage:
+  taskcluster-worker work <engine> --logging-level <level>
```

Comment:
Ok, just wondering.  I like the `taskcluster-worker work` piece, but just didn't know why engine was a like this when it's just another argument to "work".  I suppose it matters when we define what we want to be passing in from the command line to run the worker.  Do we see us also passing a list of plugins or load, or some other options? If so, specifying the engine with "--engine" might make sense.